applications/EmpireApplication/tests/sdof_static_solver/naca0012_test_probe1_ref.dat

```diff
@@ -0,0 +1,3 @@
+# Results for "element" with Id # 21 at position: x: 1; y: 0.5; z: 0
+# time VELOCITY_POTENTIAL
```

Comment:
we are now testing the velocity potential using a json file. You can check it in the tests of the compressible potential flow application